helmfile.d/charts/prometheus-servicemonitor/templates/kube-vip-monitor.yaml

```
@@ -0,0 +1,36 @@
+{{- if .Values.kubeVIP.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-servicemonitor.fullname" .) "kube-vip" | trunc 63 | trimSuffix "-" }}
```

```suggestion
  name: {{ include "prometheus-servicemonitor.fullname" . | printf "%s-kube-vip" | trunc 63 | trimSuffix "-" }}
```
If you want to simplify the name templating a tiny bit.